app/main.py

```diff
@@ -4,4 +1,23 @@
+import os
+import requests
+
+
+def get_weather(api_key: str, city: str = "Paris") -> None:
+    base_url = "http://api.weatherapi.com/v1/current.json"
```

Comment:
The base URL should be defined as a constant in uppercase style to follow best practices for constants usage. This will make it easier to manage and update the URL if needed.